UnityProject/Assets/Scripts/Machines/Jukebox.cs

```diff
@@ -0,0 +1,263 @@
+﻿using Assets.Scripts.Messages.Server.SoundMessages;
+using Audio.Containers;
+using Mirror;
+using System.Collections;
+using System.Collections.Generic;
+using System.Text;
+using UnityEngine;
+
+/// <summary>
+/// A machine that plays music choosen by it's user's tastes in a cool place like a lounge or a bar.
+/// </summary>
+public class Jukebox : NetworkBehaviour, IAPCPowered
+{
+	/// <summary>
+	/// How many watts at 240 V the Jukebox uses when not in use
+	/// </summary>
+	[SerializeField]
+	private int StandByWattUsage = 5;
+
+	/// <summary>
+	/// How many watts at 240 V the Jukebox uses when it is in use
+	/// </summary>
+	[SerializeField]
+	private int InUseWattUsage = 15;
+
+	[SerializeField]
+	private SpriteHandler spriteHandler = null;
+
+	// Sprites for when the jukebox is idle, playing, damaged.
+	[SerializeField]
+	private SpriteDataSO SpriteIdle;
+
+	[SerializeField]
+	private SpriteDataSO SpritePlaying = null;
+
+	[SerializeField]
+	private SpriteDataSO SpriteDamaged;
+
+	private List<AudioSource> musics;
+
+	private SpriteRenderer spriteRenderer;
+
+	/// <summary>
+	/// The current state of the jukebox powered/overpowered/underpowered/no power
+	/// </summary>
+	[HideInInspector] public PowerStates CurrentState;
+
+	/// <summary>
+	/// The current state of the jukebox powered/overpowered/underpowered/no power
+	/// </summary>
+	[HideInInspector] private APCPoweredDevice APCConnectionHandler;
```

Comment:
Don't think you need this `[HideInInspector]` here, it's already private.

Reply:
Ah... I think I copied this from somewhere else... :)  Gonna fix it.  Thanks